spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/DataFlowIT.java

```diff
@@ -2354,6 +2354,34 @@ private void mixedSuccessfulFailedAndUnknownExecutions(String taskDescription, S
 				.map(suffix -> task.composedTaskChildTaskByLabel(suffix).get()).collect(Collectors.toList());
 	}
 
+	private void safeCleanupAllTaskExecutions(Task task) {
+		try {
+			task.cleanupAllTaskExecutions();
+		} catch (DataFlowClientException ex) {
+			if (ex.getMessage().contains("(reason: pod does not exist)") || ex.getMessage().contains("(reason: job does not exist)")) {
+				logger.warn("Unable to cleanup task executions: " + ex.getMessage());
+			}
+			else {
+				throw ex;
+			}
+		}
+	}
+
+	private void safeCleanupTaskExecution(Task task, long taskExecutionId) {
+		try {
+			task.cleanupTaskExecution(taskExecutionId);
+		}
+		catch (DataFlowClientException ex) {
+			if (ex.getMessage().contains("(reason: pod does not exist)") || ex.getMessage()
```

Comment:
We could probably put the exception handling code into a method.  That way we only have to change it in one place.   Thoughts?

Reply:
Done